src/markdown-pages/docs/managers/zos3270terminal-manager.md

```diff
@@ -137,6 +147,20 @@ The following properties are used to configure the Zos3270Terminal Manager:
 
 </details>
 
+<details>
+<summary>Select 3270 terminal outputs</summary>
+
+| Property: | 3270TerminalOutput CPS Property |
+| --------------------------------------- | :------------------------------------- |
+| Name: | zos3270.terminal.output |
+| Description: | Experimental: Saves PNG screen representations of terminal screens to the RAS |
```

Comment:
Since this property is used to choose what formats of 3270 terminal screens we want to save to the RAS, it would be worth changing the description to something along the lines of:

```
Description: Selects the representations of 3270 terminal screens to be saved to the RAS.
```

Reply:
yes, updated thank you